tests/test_metromap.py

```diff
@@ -0,0 +1,189 @@
+import io
+import unittest
+
+import pandas as pd
+
+try:
+    import networkx as nx
+except ImportError:
+    nx = None
+
+import gurobi_optimods.metromap as map
```

Comment:
Go with `from gurobi_optimods.metromap import metromap` here. `map` is a builtin python name, in general you shouldn't override it.

Reply:
Ok, I changed it now.